website_multi_theme/wizards/website_config_settings.py

```diff
@@ -14,7 +14,7 @@ class WebsiteConfigSettings(models.TransientModel):
 
     multi_theme_id = fields.Many2one(related="website_id.multi_theme_id")
 
-    @api.model
+    @api.multi
```

Comment:
I'd better remove the decorator. Since Odoo v10, no decorator means multi.

Reply:
I guessed that it's so, but kept it for the sake of explicitivity